samples/tab-mail/ts/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
```

Comment:
Looks like you haven't implemented ESBuild in this sample.
Kindly implement that as well once you add build and deploy pipeline.

Reply:
Refer this PR: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/837/files#diff-0bd14d3659cacb47d1a8cc1a9fb204c5f99ce526486fabb589cdb029377af363

Reply:
Not required for React sample.